RELEASING.md

```diff
@@ -106,16 +113,88 @@ done
 
 Check your e-mail and make sure the number of “you’ve published this module” emails matches the number you expect.
 
-## Publish the GitHub Release
+### Publish the GitHub Release
 
 Publish the GitHub release, ensuring that the tag points to the newly landed commit corresponding to release proposal `x.y.z`.
 
-## Update CHANGELOG
+### Update CHANGELOG
 
 - After releasing is done, update the [CHANGELOG.md](https://github.com/open-telemetry/opentelemetry-js/blob/master/CHANGELOG.md) and start new Unreleased label.
 - Create a new commit with the exact title: `Post Release: update CHANGELOG.md`.
 - Go through PR review and merge it to GitHub master branch.
 
-## Known Issues
+### Known Issues
 
 - The `examples/` and `getting-started/` folders are not part of lerna packages, we need to manually bump the version in `package.json`.
+
+## Versioning
+
+### Assumptions
+
+This project uses semver v2, as does the rest of OpenTelemetry.
+
+### Goals
+
+#### API Stability
+
+Once the API for a given signal (spans, logs, metrics, baggage) has been officially released, that API module will function with any SDK that has the same major version, and equal or greater minor or patch version.
+
+For example, libraries that are instrumented with `@opentelemetry/api@1.0.1` will function with SDK library `@opentelemetry/node@1.11.33` or `@opentelemetry/web@1.3.4`
+
+#### SDK Stability:
+
+Public portions of the SDK (constructors, configuration, end-user interfaces) must remain backwards compatible.
+Internal interfaces are allowed to break.
+
+### Methods
+
+#### Mature Signals
+
+API modules for mature (i.e. released) signals will be included in the `@opentelemetry/api` module.
+
+#### Immature or experimental signals
+
+API modules for experimental signals will not be included in the `@opentelemetry/api` module, and must be installed manually. API modules will be named with an `-experimental` suffix to make it abundantly clear that depending on them is at your own risk. For example, the `@opentelemetry/metrics-api-experimental` module will provide experimental access to the unfinished metrics API. NO STABILITY GUARANTEES ARE MADE.
+
+### Examples
+
+Purely for illustration purposes, not intended to represent actual releases:
+
+#### V1.0.0 Release (tracing, baggage, propagators, context)
+
+- `@opentelemetry/api@1.0.0`
```

Comment:
According to the versioning OTEP, they want each signal to have its own API package. This package would then just depend on the stable API packages and re-export them. What do we think of that idea?

@open-telemetry/javascript-maintainers @open-telemetry/javascript-approvers

Reply:
I don't really see the advantages of doing this apart for more work when releasing ?

Reply:
for sure I would split api, about the rest hard to say, for using on daily basis I would rather have them together in on repo and keep the version aligned (all except api).

Reply:
>for sure I would split api

@obecny you would have a separate package for each trace, context, metrics, etc? Or do you just mean metrics would be split?

What if we _only_ have separate API packages and remove the "main" API package?

Reply:
if we can have only api seperate and the rest that would be the easiest, otherwise I would have:
- api
- metrics (until it is stable)
- all the rest